app/models.py

```diff
@@ -0,0 +1,273 @@
+"""
+Define SQLAlchemy models.
+
+The SQLAlchemy models for the database is defined here.
+"""
+
+import datetime
+
+from flask import current_app, url_for
+from itsdangerous import (
+    TimedJSONWebSignatureSerializer as Serializer,
+    BadSignature, SignatureExpired)
+from passlib.apps import custom_app_context as pwd_context
+
+from . import db
+
+
+class Base(db.Model):
+    """
+    Base class for models.
+
+    Define the base class for the models so others can inherit from it.
+    """
+
+    __abstract__ = True
+
+    date_created = db.Column(
+        db.DateTime, default=datetime.datetime.now(), nullable=False)
+    date_modified = db.Column(
+        db.DateTime, default=datetime.datetime.now(),
+        onupdate=datetime.datetime.now(), nullable=False)
+
+    def save(self):
+        """
+        Save to database.
+
+        Save instance of the object to database and commit.
+        """
+        db.session.add(self)
+        db.session.commit()
+
+    def delete(self):
+        """
+        Delete from database.
+
+        Deletes instance of an object from database
+        """
+        db.session.delete(self)
+        db.session.commit()
+
+
+class User(Base):
+    """
+    Set up the User model.
+
+    Set up the properties of the User object and the table name too.
+    """
+
+    __tablename__ = 'users'
+    user_id = db.Column(db.Integer, primary_key=True)
```

Comment:
I didn't use the name `id` because it is a python keyword and it might cause some errors when i try to use it someplace else.